comp/api/api/utils/common.go

```diff
@@ -11,7 +11,7 @@ import (
 	"net"
 	"net/http"
 
-	"github.com/DataDog/datadog-agent/pkg/util/grpc"
+	"github.com/DataDog/datadog-agent/pkg/util/grpc/context"
```

Comment:
Simply `context` might be a bit misleading. What about `grpcContext` ?

Reply:
Fair, done !